kernel/sysctl_setter.go

```diff
@@ -73,3 +78,15 @@ func (s *SysctlSetterImpl) Unset() error {
 	}
 	return nil
 }
+
+func (s *SysctlSetterImpl) Exists() bool {
+	if !internal.FileExists(strings.ReplaceAll(s.paramName, ".", ".")) {
```

Comment:
Shouldn't this be `!internal.FileExists("/proc/sys/" + strings.ReplaceAll(s.paramName, ".", "/"))`?